dags/leanplum_export.py

```diff
@@ -0,0 +1,33 @@
+from airflow import DAG
+from airflow.models import Variable
+from datetime import datetime, timedelta
+from utils import leanplum
+
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2019, 10, 10),
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+
+with DAG('leanplum_export',
+         default_args=default_args,
+         schedule_interval='@daily') as dag:
+
+    app_id = Variable.get('LEANPLUM_FENNEC_NIGHTLY_APP_ID')
```

Comment:
Can you add these to wtmo?

Reply:
Also, I'd need these to add to our secret hiera sops repo

Reply:
I'll add them to WTMO and send them to you